src/main/java/change/Controllers/DenomsController.java

```diff
@@ -0,0 +1,36 @@
+package change;
+
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.*;
+
+@RestController
+public class DenomsController {
+
+    @RequestMapping(value="/change", method = RequestMethod.POST)
+    public ArrayList postMethod(@RequestParam Map<String, String> requestParams) {
```

Comment:
In a production environment I'd favor a more explicit approach: call out each param explicitly `@RequestParam String coinNames, @RequestParam String coinValues, @RequestParam Double totalAmount`